HubSpot.NET/Api/Deal/HubSpotDealApi.cs

```diff
@@ -99,6 +99,46 @@ public DealListHubSpotModel<T> List<T>(bool includeAssociations, ListRequestOpti
             return data;
         }
 
+
+        /// <summary>
+        /// Gets a list of deals associated to a hubSpot Object
+        /// </summary>
+        /// <typeparam name="T">Implementation of DealListHubSpotModel</typeparam>
+        /// <param name="includeAssociations">Bool include associated Ids</param>
+        /// <param name="hubId">Long Id of Hubspot object related to deals</param>
+        /// <param name="objectName">String name of Hubspot object related to deals (contact\account)</param>
+        /// <param name="opts">Options (limit, offset) relating to request</param>
+        /// <returns>List of deals</returns>
+        public DealListHubSpotModel<T> AssociatedList<T>(bool includeAssociations, long hubId, ListRequestOptions opts = null, string objectName = "contact") where T : DealHubSpotModel, new()
```

Comment:
Only minor, but for consistency this would be named `ListAssociated` to match the equivalent API for Engagements etc.